src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure implements AreaCalculetion {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(double firstLeg, double secondLeg, String color, String name) {
+        this.secondLeg = secondLeg;
+        this.firstLeg = firstLeg;
+    }

```

Comment:
The 'color' and 'name' parameters are not used in the constructor to set the corresponding properties of the superclass 'Figure'. You should pass them to the superclass constructor using 'super(color, name)'.